routing.go

```diff
@@ -626,9 +634,8 @@ func (dht *IpfsDHT) findProvidersAsyncRoutine(ctx context.Context, key multihash
 		},
 	)
 
-	if ctx.Err() == nil {
-		// refresh the cpl for this key after the query is run
-		dht.routingTable.ResetCplRefreshedAtForID(kb.ConvertKey(string(key)), time.Now())
+	if err != nil && ctx.Err() == nil {
```

Comment:
this now isn't running if there's an error. is that functional difference intended?

Reply:
Yes, we don't want marked the Routing Table as "refreshed" by reason of running this query if the query wasn't successful.